### DISCUSSION TITLE
Categories for server

### DISCUSSION THREAD

**Discussion by kerpsonn** [pelican-dev/panel#484] [Ideas]
I think it's very good feature. Administrators could create categories for servers e.g.: "Minecraft Servers" and assign servers under them.
This would allow a clearer overview of the servers in the panel.

**Comment by rmartinoscar** [comment]
Hey the panel already [allows for server grouping/sorting in the admin area](https://github.com/pelican-dev/panel/commit/6692942f6f8f12f7a1d427237f24b40d1eeb6ba3)
![image](https://github.com/user-attachments/assets/b97db6b2-9e9e-4a82-9d4c-7d7f91efb685)
![image](https://github.com/user-attachments/assets/d3d9392e-da7e-4645-820f-45c3d13ce1c6)

Tags will surely make this even better too !
For the client area please refer to https://github.com/pelican-dev/panel/issues/353

**Comment by Boy132** [comment]
Like Martin mentioned this is already planned with Tags and the client area rework.
